redisinsight/ui/src/pages/workbench/constants.ts

```diff
@@ -27,6 +27,40 @@ export const VIEW_TYPE_OPTIONS = [
 export const getViewTypeOptions = () =>
   [...VIEW_TYPE_OPTIONS]
 
+
+export const SEARCH_COMMANDS = ['ft.search', 'ft.aggregate']
+export const GRAPH_COMMANDS = ['graph.query']
+
+const ALLOWED_PROFILE_COMMANDS = [...SEARCH_COMMANDS, ...GRAPH_COMMANDS]
+
+export function isCommandAllowedForProfile(cmd: string) {
```

Comment:
we use arrow functions almost anywhere instead of function declaration , can you change it?
also, could you please add unit tests to each functions?